### ISSUE TITLE
CPX library doesn't work on CPX (recursion depth hit)

### ISSUE THREAD

**Issue by tannewt** [adafruit/circuitpython#8053]
https://forums.adafruit.com/viewtopic.php?p=974225#p974225

**Comment by bill88t** [comment]
Pystack again?

**Comment by RetiredWizard** [comment]
I enabled MICROPY_STACKLESS which resulted in the same "RuntimeError: maximum recursion depth exceeded" message but without the traceback.

I then disabled CIRCUITPY_FULL_BUILD, CIRCUITPY_COUNTIO and CIRCUITPY_BITBANGIO and enabled CIRCUITPY_AUDIOIO and CIRCUITPY_PIXELBUF and the example scripts from the forum post ran without crashing.

I know this isn't a solution but figured it might at least help zero in on the issue.
